src/browzine-summon-adapter.js

```diff
@@ -98,56 +124,103 @@ angular.module("summonApp.directives")
     return coverImageUrl;
   };
 
-  function buildTemplate(data, browzineWebLink, bookIcon) {
-    var assetClass = "";
+  function buildTemplate(data, browzineWebLink) {
+    var wording = "";
+    var browzineWebLinkText = "";
+    var bookIcon = "https://s3.amazonaws.com/thirdiron-assets/images/integrations/browzine_open_book_icon.png";
 
-    // Customize the naming conventions for each type of item - Journal/Article - by changing the wording in the quotes below:
-    // E.g. You can customize "View the Journal" and "View Complete Issue".
     if(isJournal(data)) {
-      assetClass = "View the Journal";
+      wording = browzine.journalWording || "View the Journal";
+      browzineWebLinkText = browzine.journalBrowZineWebLinkText || "Browse Now";
     }
 
     if(isArticle(data)) {
-      assetClass = "View Complete Issue";
+      wording = browzine.articleWording || "View Complete Issue";
+      browzineWebLinkText = browzine.articleBrowZineWebLinkText || "Browse Now";
     }
 
-    // You can change the underlined "Browse Now" link name on line 122 below.
     var template = "<div class='browzine'>" +
-                     "{{assetClass}}: <a class='browzine-web-link' href='{{browzineWebLink}}' target='_blank' style='text-decoration: underline; color: #333;'>Browse Now</a> " +
+                     "{{wording}}: <a class='browzine-web-link' href='{{browzineWebLink}}' target='_blank' style='text-decoration: underline; color: #333;'>{{browzineWebLinkText}}</a> " +
```

Comment:
Worth tossing out that you can do ES5 friendly multiline string by adding a `\` to the end of each line, though the [Airbnb style guide](http://airbnb.io/javascript/#strings--line-length) doesn't seem to like multiline strings at all.

Reply:
Yup, tested that out as well. IIRC, there was some issue during testing or maybe IE, can't remember the details, but I think some `\n` newline characters were being injected or something. Settled on 'ol faithful here, hehe.